.github/workflows/build.yml

```diff
@@ -83,6 +83,9 @@ jobs:
     runs-on: ${{ matrix.platform }}
     timeout-minutes: 15
 
+    env:
+      RUSTFLAGS: -Crelocation-model=static -Dwarnings
```

Comment:
I think another way to solve this would be to move the `rustflags` config key in `config.toml` from the `build` section to the `target.x86_64-unknown-none` section. IIRC this works because target-specfic values get merged with the non-target-specific flags instead of replacing them.

Reply:
I haven't tried it, but according to [the docs](https://doc.rust-lang.org/cargo/reference/config.html#buildrustflags) it probably won't work:

> There are four **mutually exclusive** sources of extra flags. They are checked in order, with the **first one being used**:
>
> 1. CARGO_ENCODED_RUSTFLAGS environment variable.
> 2. RUSTFLAGS environment variable.
> 3. All matching target.<triple>.rustflags and target.<cfg>.rustflags config entries joined together.
> 4. build.rustflags config value.

(emphasis mine)

To me it sounds like cargo uses `RUSTFLAGS` if set and doesn't continue to look for other sources.

Reply:
Yup, you're right, nvm.